src/components/MonthlyStories/Stories.js

```diff
@@ -35,15 +44,17 @@ function Stories({ classes, stories }) {
   return (
     <div className={classes.root}>
       <div className={classes.stories}>
-        {stories.slice(0, 3).map((story, index) => (
+        {stories.map((story, index) => (
           <NewsCard
             key={story.title}
             title={story.title}
             image={story.image}
             date={story.date}
             link={story.link}
             classes={{
-              root: classNames({ [classes.newsCardMargin]: index > 0 })
+              root: classNames(classes.newsCard, {
+                [classes.newsCardMargin]: index > 0
```

Comment:
What happened to good ol `:first-child` 😢  ?

Reply:
When you right code in the flow _without_ taking a minute to think about it... This is what happens @karimkawambwa.